### PR TITLE
Add additional search tags for interface styles and pvm plugins

### LINKED ISSUE #5819: Plugin search will not find plugins when the term is in the description.
For instance, searching "Jad" will not display the Fight Caves plugin, despite the description being: "Show what to pray against Jad"

### PR DESCRIPTION
**Pull request by GravitySalad**
Closes #5819

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/fightcave/FightCavePlugin.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
 @PluginDescriptor(
 	name = "Fight Cave",
 	description = "Show what to pray against Jad",
-	tags = {"bosses", "combat", "minigame", "overlay", "prayer", "pve"}
+	tags = {"bosses", "combat", "minigame", "overlay", "prayer", "pve", "jad"}
```
[the remark this answers is not in the document]
Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/fightcave/FightCavePlugin.java]: #5819 was opened to track this

**Comment by GravitySalad** [comment]
Just to note I used to the same pattern for searching descriptions that were used for names. Might need to be changed in the future depending on the length of plugin descriptions and the similarities between others

**Comment by deathbeam** [comment]
Can you squash 1st and 2nd commit together?

**Comment by GravitySalad** [comment]
> Can you squash 1st and 2nd commit together?

should be good now

**Pull request merged**